Go install instructions are incomplete

The README prompts you to install `goenv`, but that's not sufficient for the `go` command when invoked from the AGC project directory to actually be the right version of Go as specified in `.go-version`. When all I do is install `goenv` and  go to the project directory and run `make init`, I get:

```
make init
go env -w GOPROXY=direct
goenv: version '1.17.2' is not installed (set by /public/home/anovak/build/amazon-genomics-cli/.go-version)
make: *** [init] Error 1
```

That particular version of Go has to also already be installed as an available version for `goenv` to dispatch to:

```
goenv install $(cat .go-version)
```

So that command needs to be added to the README's setup instructions.

--- a/README.md
+++ b/README.md
@@ -37,6 +37,17 @@ The Amazon Genomics CLI is written in Go.
 To manage and install Go versions, we use [goenv](https://github.com/syndbg/goenv). Follow the installation
 instructions [here](https://github.com/syndbg/goenv/blob/master/INSTALL.md).
 
+Once goenv is installed, use it to install the version of Go required by the
+Amazon Genomics CLI build process, so that it will be available when the build
+process invokes goenv's `go` shim:
+
+```bash
+goenv install $(cat .go-version)
+```
+
+You will need to do this step again whenever the required version of Go is
+changed.
+
 #### Node
 
 Amazon Genomics CLI makes use of the AWS CDK to deploy infrastructure into an AWS account. Our CDK code is written in TypeScript.

--- a/README.md
+++ b/README.md
@@ -42,7 +42,7 @@ Amazon Genomics CLI build process, so that it will be available when the build
 process invokes goenv's `go` shim:
 
 ```bash
-goenv install $(cat .go-version)
+goenv install
 ```
 
 You will need to do this step again whenever the required version of Go is

